content/v3/pulls/comments.md

```diff
@@ -62,7 +62,7 @@ Name | Type | Description
 Name | Type | Description
 -----|------|--------------
 `body`|`string` | **Required**. The text of the comment
-`commit_id`|`string` | **Required**. The SHA of the commit to comment on.
+`commit_id`|`string` | **Required**. The SHA of the latest commit of the pull request to comment on.
```

Comment:
I also interpret the position as not the line number of the file being commented, but the line number in the diff of the file being commented.

If it is the line in the diff, then that is a bit hard to figure out when using the API. Perhaps the files response can be extended to help translating a line number in the file to a line number in the diff.

I will use the API some more and update the PR.